Doc/library/sqlite3.rst

```diff
@@ -1236,12 +1299,12 @@ Connection objects
    .. attribute:: isolation_level
 
       This attribute controls the :ref:`transaction handling
-      <sqlite3-controlling-transactions>` performed by :mod:`!sqlite3`.
+      <sqlite3-transaction-control-isolation-level>` performed by :mod:`!sqlite3`.
       If set to ``None``, transactions are never implicitly opened.
       If set to one of ``"DEFERRED"``, ``"IMMEDIATE"``, or ``"EXCLUSIVE"``,
       corresponding to the underlying `SQLite transaction behaviour`_,
       implicit :ref:`transaction management
-      <sqlite3-controlling-transactions>` is performed.
+      <sqlite3-transaction-control-isolation-level>` is performed.
 
       If not overridden by the *isolation_level* parameter of :func:`connect`,
       the default is ``""``, which is an alias for ``"DEFERRED"``.
```

Comment:
Likewise, it seems you should at least mention here:

* That using the `autocommit` attribute is recommended over this, otherwise users might not be aware of it
* The interaction with this attribute and autocommit, if the latter is not set to its default value (doesn't that change or negate the behavior of this one?)